test/services/MessagePreviewServiceTest.ts

```diff
@@ -85,4 +115,28 @@ describe("MessagePreviewService", () => {
 			sandbox.restore();
 		});
 	});
+
+	describe("wasSentByABot()", () => {
+		let message: Message;
+		let discordMock: MockDiscord;
+		let messagePreview: MessagePreviewService;
+
+		beforeEach(() => {
+			discordMock = new MockDiscord();
+			message = discordMock.getMessage();
+			messagePreview = MessagePreviewService.getInstance();
+		});
+
+		it("should return true if message's author is a bot", () => {
+			message.author.bot = true;
+
+			expect(messagePreview.wasSentByABot(message)).to.be.true;
+		});
+
+		it("should return false if message's authro isn't a bot", () => {
```

Comment:
spelling mistake :)

Reply:
The authro of this pull request must be mad